.forge-snapshots/removeLiquidity.snap

```diff
@@ -1 +1 @@
-121033
\ No newline at end of file
+123509
```

Comment:
so all the snaps for add/remove liquidity are up like 2.5K 🥴 any idea why?

Reply:
grrr looks like PoolModifyLiquidity is reliant on the extsload library, sort of what Daniel observed with exttload + currencyDelta